batchlib/analysis/cell_level_analysis.py

```diff
@@ -0,0 +1,91 @@
+from tqdm.auto import tqdm
+import torch
+import numpy as np
+import h5py
+import pickle
+from functools import partial
+from concurrent import futures
+from ..base import BatchJob
+
+
+def load_sample(path, cell_seg_key, device='cpu'):
+    with h5py.File(path, 'r') as f:
```

Comment:
could you use `batchlib.utils.open_file` instead, just like so:
```
with open_file(path, 'r') as f:
   ...
```
it will do the same, but has to advantages:
- works with n5 or zarr files as well
- retries opening if the h5 file has a timeout (which sometimes happens and unncessary kills the job)